service/src/test/java/uk/gov/hmcts/reform/fpl/service/OrganisationServiceTest.java

```diff
@@ -80,21 +83,28 @@ void shouldReturnUsersFromOrganisationIfExistsInRefData() {
         when(organisationApi.findUsersByOrganisation(AUTH_TOKEN_ID, SERVICE_AUTH_TOKEN_ID, Status.ACTIVE, false))
             .thenReturn(usersInAnOrganisation);
 
-        Set<String> userIds = organisationService
-            .findUserIdsInSameOrganisation("AN");
+        Set<String> userIds = organisationService.findUserIdsInSameOrganisation("AN");
+
+        assertThat(userIds).containsExactlyInAnyOrder("40", "41");
+    }
+
+    @Test
+    void shouldNotCheckLocalMappingWhenOrganisationExistsInRefData() {
```

Comment:
this test is not needed, as it does not verify service functionality,  it is enough to add
verifyNoInteractions(lookupSpy);
in shouldReturnUsersFromOrganisationIfExistsInRefData